src/store/getters.ts

```diff
@@ -722,49 +720,20 @@ export const getNameIsSlashed = (state: StateIF): boolean => {
   return false
 }
 
-export const getShowCorpNum = (state: StateIF): CorpNumRequests.COLIN | CorpNumRequests.MRAS | false => {
-  // *** FUTURE: COLIN search (business lookup) should have been done on first page
-  if (ColinRequestActions.includes(getRequestActionCd(state)) && isNumberedRequestType) {
-    return CorpNumRequests.COLIN
-  }
-
-  if (ColinRequestActions.includes(getRequestActionCd(state)) && isXproRequestType) {
-    return CorpNumRequests.COLIN
-  }
```

Comment:
I combined `isNumberedRequestType` and `isXproRequestType` into `isBusinessLookupEntityTypes`, and this getter now returns True or False because those are the only options.